glossary.rst

```diff
@@ -80,13 +80,13 @@ Glossary
 
   widget
 
-    A single question, answer set, and attendant GUI elements, as rendered in an XForm compliant app such as :term:`Collect`.
+    A single question, answer set, and attendant GUI elements, as rendered in a XForm compliant app such as :term:`Collect`.
```

Comment:
"XForm" pronounced *eks-form*, so "an XForm" is correct.

Reply:
Thanks for pointing it out, I was so focused on the grammatical aspect that I oversaw the phonetical part.